cover.out

```diff
@@ -0,0 +1,198 @@
+mode: set
```

Comment:
Let's remove this and `cover.html` and ensure they're ignored by git by adding them to https://github.com/integr8ly/smtp-service/blob/master/.gitignore.

We can look at adding [coveralls](https://coveralls.io/) to this repo in a separate PR though.